releasenotes/notes/torch-runtime-b9c7f791b3964ab6.yaml

```diff
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    Allow leveraging Qiskit Runtime to execute a PyTorch Module including TorchConnector
+    in the cloud using the :class:`~qiskit_machine_learning.runtime.TorchRuntimeClient`.
```

Comment:
I changed it to more user friendly explanation. Actually, we can run purely classical torch code without the connector/quantum. I didn't want to enforce it because it doesn't speed up the training/predicting processes.